src/ctia/schemas/graphql/helpers.clj

```diff
@@ -29,9 +29,33 @@
             GraphQLUnionType
             TypeResolver]))
 
-;; Type registry to avoid any duplicates when using new-object
-;; or new-enum. Contains a map with types indexed by name
-(def default-type-registry (atom {}))
+(s/defschema TypeRegistry
+  "Type registry to ensure named GraphQL types are created exactly once.
+  Contains a map with derefable types indexed by name.
+  Use via get-or-update-type-registry."
+  (s/atom {s/Str
+           #_(IDeref graphql.*) ;; returns a graphql-related thing on deref
```

Comment:
It's possible this should always be a `graphql.schema.GraphQLType`, but I'm verifying that at the moment.